docs/vault/installation.md

```diff
@@ -256,7 +256,8 @@ RUST_LOG=info cargo run -p vault -- \
   --auto-register-with-faucet-url 'https://api.interlay.io/faucet' \
   --telemetry-url 'https://api.interlay.io/telemetry' \
   --btc-parachain-url 'wss://api.interlay.io/parachain' \
-  --network=testnet
+  --network=testnet \
+  --currency-id=dot
```

Comment:
I think this will not work. Should be `--currency-id DOT`?

Reply:
No I think we take the lowercase of the value anyway

Reply:
But the `=` did not work for me locally at least